ow_lander/launch/spawn.launch

```diff
@@ -64,12 +64,6 @@
     <arg name="load_robot_description" value="false"/>
   </include>
 
-  <!-- We do not have a robot connected, so publish fake joint states -->

```

Comment:
I understand we need this hotfix fast, but I don't understand this comment or why we originally had this joint_state_publisher. Ussama wrote these lines of code, so remind me to ask him about it when he returns.

Reply:
The change you made is legit, the **joint_state_publisher** node needs not to be published when Gazebo is launched.